backend/main.py

```
@@ -71,151 +137,332 @@ def get_spots(
+        skin=schemas.SkinInfo(
+            id=spot.skin.id,
+            name=spot.skin.name,
+            image_url=spot.skin.image_url
+        )
+    )
 
 @app.post("/spots", response_model=schemas.SpotResponse)
 def create_spot(
     spot: schemas.SpotCreate,
     current_user: Annotated[schemas.AuthorInfo, Depends(get_current_user)],
+    db: Session = Depends(get_db)
```

Ruff (0.14.7)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)